src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2269,4 +2269,55 @@ public Object executeScript(String script) {
         return getSeleniumHelper().executeJavascript(statement);
     }
 
+    /**
+     * @param place to check
+     * @return true if content of place is valid (according to validation pattern)
+     */
+    public boolean valueOfIsValid(String place) {
+        T element = getElementToRetrieveValue(place, null);
+        if (element == null) {
+            throw new SlimFixtureException(String.format("No element found for place '%s'", place));
+        }
+
+        Collection<String> attributesToCompare = Arrays.asList("name", "id");
+
+        // Check if element has all attributes set that we use to compare it with any invalid elements
+        checkElementHasAttributes(element, attributesToCompare, place);
+
+        List<T> invalidElements = findAllByCss("input:invalid");
+        return !containsElement(invalidElements, element, attributesToCompare);
```

Comment:
Why do you need the (complex) logic checking name and id attributes? I ran your acceptance check and it also passes if I just do a simple `!invalidElements.contains(element)`

Reply:
No idea why i did that. Will update.